examples/examples.html

```diff
@@ -1,15 +1,133 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Isn't this file autogenerated by `Metalsmith`? Changes here would be overwritten each time the examples are build, or am I  wrong? Ping @KaiVolland

Reply:
Probably right. We might need to take the changes and apply them to files in the master branch. @hblitza please ask @KaiVolland for directions.